easybuild/easyconfigs/n/netCDF/netCDF-4.5.0-intel-2018.00.eb

```diff
@@ -0,0 +1,35 @@
+name = 'netCDF'
+version = '4.5.0'
+
+homepage = 'http://www.unidata.ucar.edu/software/netcdf/'
+description = """NetCDF (network Common Data Form) is a set of software libraries 
+ and machine-independent data formats that support the creation, access, and sharing of array-oriented 
+ scientific data."""
+
+toolchain = {'name': 'intel', 'version': '2018.00'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+sources = ['v%(version)s.tar.gz']
+source_urls = [
+    'https://github.com/Unidata/netcdf-c/archive/'
+]
+
+dependencies = [
+    ('HDF5', '1.10.1'),
+    ('cURL', '7.55.1'),
+    ('Szip', '2.1.1'),
+]
+
+builddependencies = [
+    ('Autotools', '20170619'),
```

Comment:
Hmm, this is new, why did you add this specifically?

We don't have this in existing `netCDF` easyconfig, if you added this for good reason, we should do this in other `netCDF` easyconfigs too.

Reply:
@boegel The `netCDF-intel-2017a` easyconfig ([here](https://github.com/easybuilders/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-intel-2017a.eb)) includes this as builddependency. That's why I included it too (in newer version).

Reply:
This was done in #5077, but apparently it's not fully consistent yet... Thanks for the reply.

In any case, this looks good to go.